src/main/resources/processes/COMP2_CLOSE.bpmn

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_0nx86so" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.9.0">
+  <bpmn:process id="COMP2_CLOSE" name="CloseResult Closed" isExecutable="true">
+    <bpmn:startEvent id="StartEvent_COMP_CLOSE">
+      <bpmn:outgoing>Flow_056v15h</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:endEvent id="EndEvent_COMP_CLOSE">
+      <bpmn:incoming>Flow_0m4f0cw</bpmn:incoming>
+      <bpmn:incoming>Flow_0oxa4nq</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:userTask id="Validate_CompleteReason" name="Validate Close">
+      <bpmn:incoming>Flow_0jy3o41</bpmn:incoming>
+      <bpmn:outgoing>Flow_00cdpke</bpmn:outgoing>
+    </bpmn:userTask>
+    <bpmn:serviceTask id="Service_UpdateAllocationNote_Complete" name="Save Complete Note" camunda:expression="${bpmnService.updateAllocationNote(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey, execution.getVariable(&#34;CaseNote_CompleteReason&#34;), &#34;CLOSE&#34;)}">
+      <bpmn:incoming>Flow_1b8yjx0</bpmn:incoming>
+      <bpmn:outgoing>Flow_0j2omek</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:serviceTask id="Activity_061cyps" name="Close" camunda:expression="COMP_COMPLETE_REASON" camunda:resultVariable="screen">
+      <bpmn:incoming>Flow_14hh5in</bpmn:incoming>
+      <bpmn:incoming>Flow_056v15h</bpmn:incoming>
+      <bpmn:outgoing>Flow_0jy3o41</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:sequenceFlow id="Flow_0jy3o41" sourceRef="Activity_061cyps" targetRef="Validate_CompleteReason" />
+    <bpmn:exclusiveGateway id="Gateway_0t1r2vu" default="Flow_12kg1ao">
+      <bpmn:incoming>Flow_00cdpke</bpmn:incoming>
+      <bpmn:outgoing>Flow_14hh5in</bpmn:outgoing>
+      <bpmn:outgoing>Flow_12kg1ao</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_00cdpke" sourceRef="Validate_CompleteReason" targetRef="Gateway_0t1r2vu" />
+    <bpmn:sequenceFlow id="Flow_14hh5in" sourceRef="Gateway_0t1r2vu" targetRef="Activity_061cyps">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${valid == false}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:exclusiveGateway id="Gateway_0pqxili" default="Flow_0e42c4h">
+      <bpmn:incoming>Flow_12kg1ao</bpmn:incoming>
+      <bpmn:outgoing>Flow_1b8yjx0</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0e42c4h</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_12kg1ao" sourceRef="Gateway_0t1r2vu" targetRef="Gateway_0pqxili" />
```

Comment:
Should this flow have an expression for valid = true? I see it's currently the default flow; should we have valid false  flow as the default instead, it feels safer to have the default as false so we only progress if we are explicit in that the input === true rather than equals something not false.

Reply:
This is COMP2 and would be changed as per COMP_CLOSE.bpmn in ticket HOCS-3722..... but changed here anyway.. The logic was removed so there's no gateway in the file